l10n_it_account/models/account_tax.py

```diff
@@ -94,6 +118,26 @@ def compute_balance(
                 0
             ]["balance"]
             balance = balance and -balance or 0
+        if debit_only is True:
```

Comment:
Qui e sotto, basterebbe
```suggestion
        if debit_only:
```

Reply:
refactored

Reply:
:+1: questa parte non c'è più